docs/docs/1.0.8/documentation/http-api.md

```diff
@@ -190,6 +191,52 @@ curl http://localhost:5681/config
 }
 ```
 
+## Supported Envoy versions
+
+### Get supported by Kuma DP versions of Envoy
```

Comment:
I understand what you're saying here, but I think "List supported Envoy versions" would be clearer. We don't need to include "Kuma DP" because the docs context provides that information. And "get" outside the endpoint call is a little confusing.

Reply:
Thank you for a comment! :-) Let me go with your suggestion!

Reply:
done! :-)